models.js

```diff
@@ -34,3 +34,29 @@ exports.fetchArticle = (article_id) => {
     return rows[0];
   });
 };
+
+exports.fetchComments = (article_id) => {
+  const text =
+    "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC";
+  const values = [article_id];
+  return db.query(text, values).then(({ rows }) => {
+    if (rows.length === 0) {
+      return []
+    }
+    return rows;
+  })
+};
+
+exports.checkArticleExists = (article_id) => {
+  const text = "SELECT * FROM articles WHERE article_id = $1";
+  const values = [article_id];
+
+  return db.query(text, values).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "Id not found" });
+    }
+    return true
+  })
+};
+
```

Comment:
If the above model is similar to your model for /api/articles/:article_id then you could just use that instead